tests/AndroidXMigrationTests/Tests/DependenciesTests.cs

```diff
@@ -264,22 +264,22 @@ public void XamarinFormsTopLevelPackagesExpandToAll()
 				"Xamarin.AndroidX.ConstraintLayout.Core",
 				"Xamarin.AndroidX.CoordinatorLayout",
 				"Xamarin.AndroidX.Core",
-				//"Xamarin.AndroidX.Core.Core.Ktx",
```

Comment:
You will see those (s0metimes) if something needs to be downgraded. Sometimes it influences dependencies. Newer versions usually mean more dependencies (transitive).

Reply:
Spot-checking some of these seems to make sense.

The new version `Xamarin.AndroidX.AppCompat` added:
- Xamarin.AndroidX.Emoji2
- Xamarin.AndroidX.Emoji2.ViewsHelper

Most of the others seem to come through the new version of `SlidingPaneLayout` adding a dependency to `Window`:
- Xamarin.AndroidX.Legacy.Support.V4
  - Xamarin.AndroidX.Legacy.Support.Core.UI
    - Xamarin.AndroidX.SlidingPaneLayout
       - Xamarin.AndroidX.Window
         - Xamarin.KotlinX.Coroutines.Android
           - Xamarin.Kotlin.StdLib.Jdk8
           - Xamarin.KotlinX.Coroutines.Core.Jvm

Reply:
yes.

My point was: if I need to downgrade nuget version sometimes I will have extra dependencies and I leave those uncommented, because upgrading versions will (most likely) need those again (newer versions bring in new dependencies).